src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -393,7 +387,9 @@ private void AbortWorkflow(IProgressController controller, CancellationToken tok
 
         internal /*testing purposes*/ IEnumerable<Language> GetBindingLanguages()
         {
-            return this.BindingProjects.Select(Language.ForProject).Distinct();
+            return this.BindingProjects.Select(Language.ForProject)
+                                       .Distinct()
+                                       .Where(l => this.host.SupportedPluginLanguages.Contains(l));
```

Comment:
`l` doesn't mean anything. Let's get rid of it with a method group, or let's rename it to `language`.